src/components/Footer/index.js

```diff
@@ -0,0 +1,116 @@
+import Link from "next/link";
+import Image from "next/image";
+import facebookLogo from "../../../public/images/logos/facebook.svg";
+import linkedinLogo from "../../../public/images/logos/linkedin.svg";
+import mediumLogo from "../../../public/images/logos/medium.svg";
+import rssLogo from "../../../public/images/logos/rss.svg";
+import {
+  AddressWrapper,
+  CompanyWrapper,
+  Copyright,
+  FooterWrapper,
+  NavigationItem,
+  NavigationWrapper,
+  SocialItem,
+  SocialWrapper,
+  Title,
+} from "./Footer.styled";
+
+const Footer = () => {
+  return (
+    <FooterWrapper>
+      <CompanyWrapper>
+        <Title>
+          <span>Allcorrect</span>
+          <p>Game outsourcing studio</p>
+        </Title>
+        <NavigationWrapper>
+          <NavigationItem>
+            <Link href="/">
+              <a>Portfolio</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem>
+            <Link href="/about">
+              <a>About Us</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem>
+            <Link href="/services">
+              <a>Services</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem>
+            <Link href="/pricing">
+              <a>Pricing</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem>
+            <Link href="/blog">
+              <a>Blog</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem>
+            <Link href="/join">
+              <a>Join us</a>
+            </Link>
+          </NavigationItem>
+          <NavigationItem color="blue">
+            <Link href="/contact">
+              <a>Get in touch</a>
+            </Link>
+          </NavigationItem>
+        </NavigationWrapper>
+      </CompanyWrapper>
+      <AddressWrapper>
+        <p>
+          <span>IRELAND:</span> 66 Silken Vale, Maynooth, co. Kildare, Ireland W23 V3P2
+        </p>
+        <p>
+          <span>CANADA:</span> 119 Spadina Ave, Toronto, ON M6E3J3
+        </p>
+      </AddressWrapper>
+      <Copyright>
+        <p>© Allcorrect Group 2006—2021, «Legal information»</p>
+        <p>site@allcorrectgames.com</p>
+      </Copyright>
+      <SocialWrapper>
```

Comment:
i didnt like using position absolute for this component 
try your best to avoid using absolute positioning 

instead create container that wrapes addresswrapper copyright and social wrapper 
use flex space between
then wrap addresswrapper and copyright inside a container and use grid 

this is my suggestion and its optional but position absolute should be fixed